.github/workflows/assets.yml

```diff
@@ -47,56 +38,57 @@ jobs:
           # if the default server is responding -- we can skip apt update
           $APT_INSTALL || { sudo apt update && $APT_INSTALL ; }
           echo "ARCH=${{ matrix.arch }}" >> "$GITHUB_ENV"
-          echo "TAG=$(git describe --always --tags | grep -E '[0-9]*\.[0-9]*\.[0-9]*' || echo snapshot)" >> "$GITHUB_ENV"
-      - name: ensure clean assets dir
```

Comment:
Why keep assets folder?

Reply:
I have just reverted to the version of the file from the version https://github.com/lf-edge/eve/blob/0ee051623dcc26176c82ad012a8f29ca35fbbd9f/.github/workflows/assets.yml

Reply:
nvm, we're reverting